### PR TITLE
Delete module for today before building

### PR DESCRIPTION
**Pull request by kieranricardo**
This PR deletes the module with todays date before attempting to build the module.

For some reason my recent build didn't use the latest `dea-cogger` from https://packages.dea.ga.gov.au/dea-cogger/ so I'm re-building the module.

### CODE REVIEW

File: dags/nci_build_dea_module.py [hunk 1]
```diff
@@ -26,7 +26,8 @@
     build_env_task = SSHOperator(
         task_id=f'build_dea_module',
         ssh_conn_id='lpgs_gadi',
-        command="""
+        command=f"""
+        rm -r /g/data/v10/public/modules/dea/{datetime.datetime.now().strftime("%Y%m%d")}
```
Comment (omad) [inline dags/nci_build_dea_module.py:30]: Can you change it to use the shell to get the current day, I suspect that python code here will run in UTC and delete yesterdays module (or fail) if run before 10 or 11 in the AM.
`$(date +%Y%m%d)`

Comment (kieranricardo) [reply dags/nci_build_dea_module.py:30]: This is how `dea-orchestration` gets the date:  **https://github.com/GeoscienceAustralia/dea-orchestration/blob/19ed1ca001dd5fbb692ba4921d9552d3973fbae1/nci_environment/build_environment_module.py#L102**

I think we should probably use the same method to make sure we're deleting the same module we're building, what do you reckon?

**Review by omad — CHANGES_REQUESTED** [review]

**Comment by kieranricardo** [comment]
good point! I'll check the timezone

**Comment by kieranricardo** [comment]
you were right! The NCI uses AESDT, I'll this in

**Review by omad — APPROVED** [review]
Awesome!

Sorry I forgot to press *submit* on the review on Monday! :(

**Pull request merged**